core-services/prow/02_config/redhat-appstudio/build-definitions/_pluginconfig.yaml

```diff
@@ -0,0 +1,67 @@
+approve:
```

Comment:
I'm not sure we really need approve plugin. We removed it from build-service and image-controller in order to get rid of ci bot's spam messages on PRs